scripts/Install-VSCode.ps1

```diff
@@ -92,6 +114,14 @@
 #>
 [CmdletBinding()]
 param(
+    [parameter()]
+    [ValidateSet(,"64","32")]
```

Comment:
If you use `64bit` and `32bit`, then you can use $Architecture in the Write-Host output to indicate which bit-version is being installed.  Or I guess you could use " ... ${Architecture}-bit ..." but I think the args 64bit / 32bit read a bit nicer.